examples/pyplots/whats_new_98_4_fill_between.py

```diff
@@ -7,14 +7,14 @@
 import matplotlib.pyplot as plt
 import numpy as np
 
-x = np.arange(0.0, 2, 0.01)
-y1 = np.sin(2*np.pi*x)
-y2 = 1.2*np.sin(4*np.pi*x)
+x = np.arange(-5, 5, 0.01)
```

Comment:
Title still needs to be changed (this isn't your fault, but this name is just an error from the conversion to sphinx gallery).  

I still think that `np.sin` is just as useful an example as yours.  not sure why you think this is more illustrative - is it just because you lose track of what is y1 and what is y2?

Reply:
fwiw I prefer the parabolas.

Reply:
I'll commit the name change